client/src/main/java/org/apache/celeborn/client/ShuffleClientImpl.java

```diff
@@ -495,35 +568,30 @@ protected void limitZeroInFlight(String mapKey, PushState pushState) throws IOEx
     }
   }
 
-  private boolean waitRevivedLocation(
-      ConcurrentHashMap<Integer, PartitionLocation> map, int partitionId, int epoch) {
-    PartitionLocation currentLocation = map.get(partitionId);
+  boolean checkRevivedLocation(
```

Comment:
this method is not straightforward enough, better to add JavaDoc to explain it a bit

Reply:
done